db/migrate/20220202030935_add_columns_to_solidus_acima_payment_sources.rb

```diff
@@ -1,6 +1,5 @@
 class AddColumnsToSolidusAcimaPaymentSources < ActiveRecord::Migration[6.1]
   def change
-    add_column :solidus_acima_payment_sources, :api_key, :string
```

Comment:
Removing the `add_column` here and adding the `remove_colum` in the next commit will prevent to run the migration.